spec/clog/core_spec.clj

```diff
@@ -29,4 +29,10 @@
     (save-blog "test-blog")
       (let [id (str (:id (first (all-blogs))))]
         (should-contain "test-blog</textarea>" 
-          (:body (app {:query-params [["edit" id]]}))))))
+          (:body (app {:query-params [["edit" id]]})))))
+
+  (it "updates blog when requested"
+    (save-blog "test-blog")
```

Comment:
It's difficult to know where 'save-blog' method lives because it has no scope.  Instead of require 'all' you could do it in a way such that you have Class.function e.g. blog.save-blog..